addons/project/static/tests/views/project_task_kanban/project_task_kanban_view_tests.js

```diff
@@ -0,0 +1,88 @@
+/* @odoo-module */
+
```

Comment:
Unnecessary :)

Reply:
You can remove the one in `addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js` too

Reply:
> Unnecessary :)

saas-16.3 is the current branch target, so we will take it off the master during the forward port.

Reply:
Oh yes you're right. But then why is the branch named `master-...`?

Reply:
Because it has been retargetted. Ok :)